docs/en/latest/tutorials/health-check.md

```diff
@@ -161,3 +161,72 @@ The health check status can be fetched via `GET /v1/healthcheck` in [Control API
 curl http://127.0.0.1:9090/v1/healthcheck/upstreams/healthycheck -s | jq .
 
 ```
+
+## Health Check Information
+
+APISIX provides comprehensive health check information, with particular emphasis on the `status` and `counter` parameters for effective health monitoring. In the APISIX context, nodes exhibit four states: `healthy`, `unhealthy`, `mostly_unhealthy`, and `mostly_healthy`. The transition of a node's state depends on the success or failure of the current health check, along with the recording of four key metrics in the `counter`: `tcp_failure`, `http_failure`, `success`, and `timeout_failure`.
```

Comment:
I suggest we _explicitly_ explain `mostly_healthy` and `mostly_unhealthy` as well. For example, `mostly_healthy` status means the current node status is healthy, but APISIX starts to receive unhealthy indications during health checks.

I understand this concept is somewhat conveyed in the state transition diagram. However, I find it a bit easier to understand if you explain it up front as well. What do you guys think? @luoluoyuyu @monkeyDluffy6017